common.docs/i-bem-js/i-bem-js.ru.md

```diff
@@ -494,7 +494,7 @@ provide(BEMDOM.decl(this.name, {}));
 
 modules.define('bblock', ['i-bem__dom', 'ablock'], function(provide, BEMDOM, ABlock) {
 
-provide(BEMDOM.decl({ block : this.name, baseBlock : ABlock ));
+provide(BEMDOM.decl({ block : this.name, baseBlock : ABlock }, {}));
```

Comment:
This commit fix the second bracket
 { block : this.name, baseBlock : ABlock **}** <- this